setup.py

```diff
@@ -32,7 +32,7 @@
         # to get the version of the simulator that supports the mcm=True argument for Monte Carlo
         # simulation of mid-circuit measurement, which AutoQASM requires.
         # NOTE: This change should remain in the feature/autoqasm branch; do not merge to main.
-        "amazon-braket-default-simulator @ git+https://github.com/aws/amazon-braket-default-simulator-python.git@46aea776976ad7f958d847c06f29f3a7976f5cf5#egg=amazon-braket-default-simulator",  # noqa E501
+        "amazon-braket-default-simulator @ git+https://github.com/aws/amazon-braket-default-simulator-python.git@61ce2136e42bdac71fc3df204d4f09a83bb3aed4#egg=amazon-braket-default-simulator",  # noqa E501
```

Comment:
why don't we just point to the branch?

Reply:
I recall the reason is that mcm-sim and AutoQASM are both developing and sometime UX breaks with updates. So we pin to commit hash instead of the branch itself.